packages/select/src/Select.d.ts

```diff
@@ -1,13 +1,26 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import type { ReactNode } from 'react';
-import type { Props as RSelectProps, GroupBase } from 'react-select';
+import type { Props as RSelectProps, GroupBase, StylesConfig, ThemeConfig } from 'react-select';
 import type { AsyncPaginateProps } from 'react-select-async-paginate';
 import type { FieldValidator } from 'formik';
+import React from 'react';
+
+export interface SelectStyleProps {
+  showError?: boolean;
+  styles?: React.CSSProperties;
+  isInline?: boolean;
+}
+
+export declare const SelectStyles: (props: SelectStyleProps) => {
+  styles: StylesConfig;
+  theme: ThemeConfig;
+};
 
 export type SelectProps<Option, IsMulti extends boolean, Group extends GroupBase<Option> = GroupBase<Option>> = {
   allowSelectAll?: boolean;
   autofill?: boolean | Record<string, string | ((value: any) => any)>;
   cacheUniq?: any | any[];
+  clearButtonClassName?: string = 'btn btn-link link';
```

Comment:
I think it's ok to leave off the default value here

Reply:
Do you mean that it's ok to leave it off because it's in the Select.defaultProps or to not have a default at all? I felt a bit weird adding bootstrap to the component when we're taking it out of so many others so that's why I just made it the default.

Reply:
The first one. The actual value that the component is using is coming from `defaultProps`.  When you set a default in the type def, it's saying the default type is `btn btn-link link`, not the value.